src/ui/components/PrefixBadge.module.css

```diff
@@ -0,0 +1,88 @@
+:root:global(.theme-light) .consoleBadge,
+:root:global(.theme-dark) .consoleBadge {
+  width: 12px;
+  height: 12px;
+  border-radius: 100%;
+  margin-top: 5px;
+  margin-left: 4px;
+  margin-right: 2px;
+}
+
+:root:global(.theme-light) .pickerBadge,
+:root:global(.theme-dark) .pickerBadge {
+  border-radius: 100%;
+  position: relative;
+  height: 20px;
+  width: 20px;
+}
+
+:root:global(.theme-light) .unicorn,
+:root:global(.theme-dark) .unicorn {
+  background-image: url(~devtools/client/debugger/images/sources/unicorn-pixel.svg) !important;
+  background-size: auto 100%;
+  width: 16px;
+  height: 16px;
+  margin-top: 3px;
+  margin-left: 2px;
+  margin-right: 0;
+  border-radius: 0;
+}
+
+:root:global(.theme-light) .pickerunicorn,
+:root:global(.theme-dark) .pickerunicorn {
+  background-image: url(~devtools/client/debugger/images/sources/unicorn-pixel.svg) !important;
+  background-size: auto 100%;
+  border-radius: 0;
+  background-color: transparent;
+}
+
+:root:global(.theme-light) .empty,
```

Comment:
Yeah, so we are saying here that the color "empty" is `theme-base-90`, so that is what we are getting both in the picker and in the console messages. As I mentioned with the unicorn thing: the picker, the console, and the panel status are all pretty different in terms of how these different prefix choices interact with them, so it would probably be *best* to have separate components for each, rather than trying to make them all work with exactly the same styles. However, if you wanted to stick with this approach, I would suggest making `empty` equivalent to a `transparent` background color rather than `theme-base-90`. That means the picker will *actually* be empty (meaning the circle will be white and thus invisible), but that is sort of the trade-off we have to make if we insist that each place these choices show-up has to use the same styles.